WasatchNET/OceanSpectrometer.cs

```diff
@@ -22,7 +22,7 @@ internal OceanSpectrometer(UsbRegistry usbReg, int index = 0) : base(usbReg)
             int errorReader = 0;
             integrationTime_ = SeaBreezeWrapper.seabreeze_get_min_integration_time_microsec(specIndex, ref errorReader) / 1000;
             if (errorReader != 0)
-                integrationTime_ = 8000;
+                integrationTime_ = 80;
```

Comment:
So integrationTime_ is in milliseconds, and you're defaulting to 80ms?  Okay.  This is one reason I try to always embed units in variable names where possible.